package.json

```
@@ -30,8 +30,8 @@
     "cookies-eu-banner": "2.0.1",
     "cssnano": "4.1.10",
     "del": "5.1.0",
-    "easymde": "2.9.1-246.0",
-    "gulp": "4.0.2",
+    "easymde": "2.9.1-299.0",
+    "gulp": "^4.0.2",
```

```suggestion
    "gulp": "4.0.2",
```

Je crois qu'on avait décidé de mettre des versions fixes, non ? :) #5458

---

C'est moi qui est changé ça ?

---

C'est Git qui s'est perdu car j'ai modifié cette ligne dans #5672. Donc ce n'est pas de ta faute, mais c'est à corriger :)